### PR TITLE
Quick & dirty setup of a read only airbyte instance

### PR DESCRIPTION
**Pull request by michel-tricot**
## What
Quick & dirty setup of a read only airbyte instance

## How
We expose airbyte through a public LB and we disable all the `write` api calls (accessible via [demo.airbyte.io](https://demo.airbyte.io))
We expose an admin version of airbyte through a tunnel so we can update configs

I wanted to do it on GCP and use the Identity Aware Proxy to have some basic auth and enable admin through the same lb but the frontend currently doesn't propagate cookies in api calls (see https://github.com/airbytehq/airbyte/issues/1756)

I have a poor memory and I don't what to have to go through the AWS console nor try to remember how to create a tunnel so:

To establish the tunnel:
```
./tools/internal/demo.sh tunnel
```

To ssh on the instance:
```
./tools/internal/demo.sh ssh
```

We can update the configuration with terraform (through terraform cloud)

### CODE REVIEW

File: terraform/aws/demo/core/main.tf [hunk 1]
```diff
@@ -0,0 +1,50 @@
+data "aws_security_group" "default-sg" {
+  id = var.default-sg
+}
+
+data "aws_ami" "amazon-linux-2" {
+  owners = [137112412989]
```
Comment (cgardens) [inline terraform/aws/demo/core/main.tf:6]: what is this magic number?

**Review by cgardens — COMMENTED** [review]
started looking at this but going to need more time to go through in detail. putting it down for now.

File: tools/internal/demo.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -e
+
+. tools/lib/lib.sh
+
+TUNNEL_PORT=${TUNNEL_PORT:-4242}
+SSH_KEY=${SSH_KEY:-~/.ssh/airbyte-app.pem}
```
Comment (cgardens) [inline tools/internal/demo.sh:8]: is this ssh key shared somewhere?

**Review by cgardens — APPROVED** [review]
looks good.

File: tools/internal/demo.sh (hunk 2 above)
Comment (michel-tricot) [reply tools/internal/demo.sh:8]: now it is. In lastpass shared-engineering

**Pull request merged**